### DISCUSSION TITLE
Exploring recipes with LLMs, Ollama and R – Thomas Sandmann’s blog

### DISCUSSION THREAD

**Discussion by giscus** [tomsing1/blog#10] [Announcements]
# Exploring recipes with LLMs, Ollama and R – Thomas Sandmann’s blog

https://tomsing1.github.io/blog/posts/vectorsearch/

**Comment by Wang-Yong2018** [comment]
So intereested article about using R do RAG.  Learning a lot from the article, especially abut the interacting with ollama from R part.
 
One thought for reference, Regarding  using RAG In Production environment, I think using database and ETL process , the history data embedding result  should be more reproduciable and  incrementally conduct.  
Recently, I use PGAI to demo  RAG for a pizza customer voice case as well. 
https://github.com/Wang-Yong2018/pgai/blob/main/docs/pgai_rag_demo_why_customer_donot_like_pizza.md

**Reply by tomsing1** [reply]
Thanks for commenting and sharing your code! Learning how to store & retrieve embeddings using databases is next on my to-do list.
